<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>774</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Eye-Tracking Visualizer</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1262</width>
      <height>754</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="plotting_row_hbox">
     <item>
      <layout class="QVBoxLayout" name="plotting_row_options_vbox">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QScrollArea" name="error_scroll_area">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>325</width>
           <height>46</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>325</width>
           <height>46</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scroll_area_widget_contents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>323</width>
            <height>44</height>
           </rect>
          </property>
          <widget class="QLabel" name="error_message">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>325</width>
             <height>46</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>325</width>
             <height>46</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>325</width>
             <height>46</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="directory_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Import Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="directory_vspacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="stimulus_hbox">
         <item>
          <widget class="QLabel" name="stimulus_label">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Stimulus:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="stimulus_menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="participant_select_deselect_all_button_hbox">
         <item>
          <widget class="QPushButton" name="participant_select_all_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>21</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="participant_select_deselect_all_hspacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="participant_deselect_all_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>82</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>82</width>
             <height>21</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Deselect All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="participant_selection_menu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>325</width>
           <height>350</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>325</width>
           <height>350</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="participant_selection_widget_holder">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>323</width>
            <height>348</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="data_type_hbox">
         <item>
          <widget class="QLabel" name="data_type_label">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Data Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="data_type_menu">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="analysis_type_hbox">
         <item>
          <widget class="QLabel" name="analysis_type_label">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Analysis Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="analysis_type_menu">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="plot_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="bottom_vspacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="plotting_row_vspacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="plotting_row_plot_vbox">
       <item>
        <widget class="QWidget" name="plot_placeholder" native="true">
         <property name="minimumSize">
          <size>
           <width>900</width>
           <height>700</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>900</width>
           <height>749</height>
          </size>
         </property>
         <zorder></zorder>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="min_samples_label">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;min_samples &lt;br/&gt;per cluster&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="min_samples_slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
